src/main/java/net/mcreator/ui/dialogs/imageeditor/ResizeDialog.java

```diff
@@ -66,6 +77,8 @@ public ResizeDialog(MCreator window, Canvas canvas, Layer layer, VersionManager
 		constraints.add(height);
 		constraints.add(L10N.label("dialog.imageeditor.resize_enable_anti_aliasing"));
 		constraints.add(type);
+		constraints.add(L10N.label("dialog.imageeditor.resize_canvas"));

```

Comment:
I would add a new translation for this as this text can be misleading (it doesn't always resize the canvas).

Reply:
:heavy_check_mark: Fixed